libseccomp/src/filter_context.rs

```diff
@@ -670,6 +833,19 @@ impl ScmpFilterContext {
     ///
     /// If this function is called with an invalid filter, an issue is encountered
     /// setting the attribute, or the libseccomp API level is less than 4, an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// let mut ctx = ScmpFilterContext::new_filter(ScmpAction::Allow)?;
+    /// if check_api(4, ScmpVersion::from((2, 5, 0))).unwrap() {
+    ///     ctx.set_ctl_ssb(true)?;
+    /// } else {
+    ///     assert!(ctx.set_ctl_ssb(true).is_err());
+    /// }
```

Comment:
```suggestion
    /// if check_api(4, ScmpVersion::from((2, 5, 0))).unwrap() {
    ///     ctx.set_ctl_ssb(false)?;
    /// } else {
    ///     assert!(ctx.set_ctl_ssb(false).is_err());
    /// }
```

Reply:
Added the fix in [commit](https://github.com/libseccomp-rs/libseccomp-rs/commit/2e07e17f07f92e2b6f7766c3c2d67495dc762efe)